CardanoSharp.Wallet.Test/TransactionTests.cs

```diff
@@ -38,7 +38,121 @@ public TransactionTests()
         [Fact]
         public void DeserializeTransaction()
         {
+            //input & output
+            var input1Addr = "98035740ab68cad12cb4d8281d10ce1112ef0933dc84920b8937c3e80d78d120".HexToByteArray();
```

Comment:
Small nitpick since this is a test - would this be more appropriately named input1TxHash?